apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -215,24 +216,43 @@ export class AdministratorResourcesActionBlock extends Component {
 
 export class SpecialAnnouncementActionBlock extends Component {
   static propTypes = {
-    hocLaunch: PropTypes.string
+    announcement: shapes.specialAnnouncement
   };
 
+  state = {
+    buttonList: this.createButtonList()
+  };
+
+  createButtonList() {
+    const buttonList = [];
+    const {announcement} = this.props;
+    buttonList.push({
+      id: announcement.buttonId,
+      url: announcement.buttonUrl,
+      text: announcement.buttonText
+    });
+    if (
+      announcement.buttonId2 &&
+      announcement.buttonUrl2 &&
+      announcement.buttonText2
```

Comment:
I think the spec said button id was not required - it might not be provided if we're not trying to set up tracking on the button.  Let's render if url && text are provided.